src/app/app.module.ts

```diff
@@ -88,6 +94,8 @@ export function tokenGetter(): string | null {
       useClass: WithCredentialsInterceptor,
       multi: true,
     },
+    { provide: API_URL, useValue: `${environment.api.protocol}://${environment.api.baseUrl}` },
+    { provide: LOCALE_ID, useValue: 'de-de' },
```

Comment:
Wozu wird das verwendet?

Reply:
API_URL: Damit kann man die api url per DI in die Services injizieren und muss nicht in jedem Service die Url zusammenbauen. Schau mal in den UserService, da wird das verwendet.

LOCALE_ID: Das musste ich einfügen, damit das Datum nach deutschem Format gerendert wird in der Liste